ui/packages/catalog-ui-search/src/main/webapp/component/search-form/forms-system/search-form-system-tab-container.view.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+/* global require */
+import React from 'react'
+import styled from '../../../react-component/styles/styled-components'
+const Marionette = require('marionette')
+const $ = require('jquery')
+const SystemSearchFormCollectionView = require('./search-form-system.collection.view')
+const SearchFormSystemCollection = require('./search-form-system-collection-instance')
+const CustomElements = require('js/CustomElements')
+const LoadingCompanionView = require('component/loading-companion/loading-companion.view')
+const Router = require('component/router/router')
+
+const Root = styled.div`
+  > .title {
+    text-align: center;
+    font-size: 20px;
+    padding: 15px;
+  }
+  > .collection {
+    margin: auto;
+    max-width: 1020px;
+    height: 100%;
+    overflow: auto;
+  }
+`
+module.exports = Marionette.LayoutView.extend({
+  className: 'customElement',
+  regions: {
+    collection: '.collection',
+  },
+  initialize: function() {
+    this.searchFormSystemCollection = SearchFormSystemCollection
+    this.listenTo(
+      this.searchFormSystemCollection,
+      'change:doneLoading',
+      this.handleLoadingSpinner
+    )
+  },
+  template() {
+    return (
+      <Root>
+        {Router.attributes.path === 'forms(/)' ? (
+          <React.Fragment>
+            <div className="title">
+              {' '}
+              These are system search forms and <b>cannot be changed</b>{' '}
+            </div>
+            <div className="collection" />{' '}
```

Comment:
✏️ 	If you pull this `div` outside I think you can also get rid of the `React.Fragment` wrapper

Reply:
It needs to have the React.Fragment wrapper if the ternary operator is being used in `<Root>`